BossMod/BossModule/AIHints.cs

```diff
@@ -64,6 +68,7 @@ public class Enemy(Actor actor, bool shouldBeTanked)
     // clear all stored data
     public void Clear()
     {
+        ForceMovementIn = float.MaxValue;
```

Comment:
i need to fix this, this is the wrong place for it. it tends to stick when you disable ai mode